src/App.vue

```diff
@@ -450,9 +455,9 @@ export default class App extends Mixins(
       console.log('Error fetching user info or updating Launch Darkly =', error)
     }
 
-    // now that LaunchDarkly has been updated (ie, in case of user targeting),
-    // check whether to use this Entity Dashboard or the new Business Dashboard
-    if (GetFeatureFlag('use-business-dashboard') && (this.$route.name === Routes.DASHBOARD)) {
+    // check whether to redirect to the new Business Dashboard
+    if (this.$route.query.noRedirect !== undefined) this.setNoRedirect(true)
+    if ((this.$route.name === Routes.DASHBOARD) && !this.isNoRedirect) {
```

Comment:
We no longer need the FF but I did this so we can still sort-of see the original entity dashboard. And I save it to the Store so that links work even if the original route (eg, dashboard) is changed.

Otherwise, I'd have to append "noRedirect=true" to all route changes.